score/etl/sources/calenviroscreen/etl.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+
+from etl.base import ExtractTransformLoad
+from utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class CalEnviroScrenETL(ExtractTransformLoad):
+    def __init__(self):
+        self.CALENVIROSCREEN_FTP_URL = "https://justice40-data.s3.amazonaws.com/CalEnviroScreen/CalEnviroScreen_4.0_2021.zip"
+        self.CALENVIROSCREEN_CSV = self.TMP_PATH / "CalEnviroScreen_4.0_2021.csv"
+        self.CSV_PATH = self.DATA_PATH / "dataset" / "calenviroscreen4"
+
+        # Definining some variable names
+        self.CALENVIROSCREEN_SCORE_FIELD_NAME = "calenviroscreen_score"
```

Comment:
I'm wondering if we want to move these declarations outside of `init`, so that they can be accessed using autocomplete in an IDE as, e.g., `CalEnviroScrenETL.CALENVIROSCREEN_SCORE_FIELD_NAME` from another class that is trying to point to the same field.

Reply:
I haven't seen that pattern across classes, but will keep an eye out on it.